samples/snippets/src/main/java/com/example/bigtable/Filters.java

```diff
@@ -354,6 +359,26 @@ public static void filterComposingCondition(String projectId, String instanceId,
   // [END bigtable_filters_composing_condition]
   // [END_EXCLUDE]
 
+  // [START bigtable_hw_get_with_filter]
+  private static void readRowFilter(
+      String projectId, String instanceId, String tableId, Filter filter) {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
```

Comment:
I think we can remove the second line here since try with resources will automatically, call close at the end of the block!

Reply:
Sure. Thanks